services/minespace-web/webpack.config.ts

```diff
@@ -38,10 +38,11 @@ const PATHS = {
 };
 
 const BUILD_FILE_NAMES = {
-  css: "style/[name].[contenthash:4].css",
-  bundle: "js/bundle.[chunkhash:4].js",
-  vendor: "js/[id].[chunkhash:4].js",
-  assets: "assets/[name].[hash:4].[ext]",
+  css: "style/[name].[contenthash:8].css",
+  cssChunkName: "style/[name].[chunkhash:8].css",
```

Comment:
Changing these from `chunkhash:4` / `chunkhash:8` fixed the webpack build issue that popped up 🤷